### PR TITLE
Require drush 9.4.0 and use' --existing-config'

### PR DESCRIPTION
**Pull request by unclegcb**
#18 
Not sure we really want to lock everyone into Drush 9.4.0 but here this is for discussion.

### CODE REVIEW

**Comment by mariacha** [comment]
I'm not sure about forcing the upgrade to drush 9.x either, although it seems like we'll need to do it pretty soon...

My major issue is with them changing how aliases work. But that's the direction things are moving, so we'll need to figure it out.

**Comment by unclegcb** [comment]
Is that why drush sa fails when I am in those projects?? That makes sense.

Seems like we could use this, no? https://drushcommands.com/drush-9x/site/site:alias-convert/

**Comment by mariacha** [comment]
Yes we could. I was dithering on when we'd do it though -- during `robo config` maybe?

**Comment by mariacha** [comment]
I threw something in the `robo configure` command -- this is not tested. Maybe you can see if it works @unclegcb?

**Comment by unclegcb** [comment]
`drush site:alias-convert` only converts local alias files. It also requires a destination directory argument with a full path.

If we are using drush aliases to talk to Pantheon, we'll need to have everyone do some updating on their local, including installing this plugin manually:
https://github.com/pantheon-systems/terminus-aliases-plugin

And the following these instructions to update pantheon site aliases:
https://pantheon.io/blog/drush-940-release-available-pantheon

File: composer.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     },
     "require": {
         "vlucas/phpdotenv": "^2.2",
-        "consolidation/robo": "^1.1.0"
+        "consolidation/robo": "^1.3",
+        "drush/drush": "9.4.0"
```
Comment (wxactly) [inline composer.json]: Unless there's a reason to lock this exact version, we should use semvar here instead:
```json
"drush/drush": "^9.4"
```

Comment (unclegcb) [reply composer.json]: @wxactly sadly, there is https://github.com/thinkshout/robo-drupal/issues/18#issuecomment-436047051

Comment (wxactly) [reply composer.json]: Haha lovely... just lovely. Welp, had to check 🤷‍♂️

**Comment by mariacha** [comment]
I'm going to merge this one as well.

**Pull request merged**